src/BookChapterVerseAnswers.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import 'bulma/css/bulma.min.css';
+import { Heading, Tile, Section, Box, Notification } from 'react-bulma-components';
+import BookAnswers from './BookAnswers';
+import VerseAnswers from './VerseAnswers';
+
+const BookChapterVerseAnswers = ({ book_answers, chapter_answers, verse_answers }) => {
+
+  const [points, setPoints] = useState(0);
+  const [showAnyThing, showAny] = useState(true);
```

Comment:
@dinushkaherath @JCTalbott this looks pretty great!  

Some feedback.  Here in the useState section, let's use the convention const [thing, setThing] so lines 10-13 would change.  line 10 for example:
```const [showAnything, setShowAnything] = useState(true);```
this just matches convention and makes it easy to understand when the setter is used elsewhere.

Now, regarding handleClick, this might be my weird quirkiness, but I think I would rather **begin** with separate click handlers for each set of answers.  So define handleBookAnswerClick, handleVerseAnswerClick, handleVerseNumberClick, each quite small and focused, and pass them to the subcomponents.  I think it will make the whole thing easier to reason about.